pkg/machinery/kernel/kernel.go

```diff
@@ -31,6 +31,14 @@ var DefaultArgs = []string{
 	"ima_template=ima-ng",
 	"ima_appraise=fix",
 	"ima_hash=sha512",
+	// TODO: configure it properly
+	"talos.selinux=permissive",
```

Comment:
this is a default set of args which applies to any version of Talos, we should not probably modify it that aggressively.

we can modify per-test if we want to, but probably we should split it per-version

Reply:
We could do something like `security=selinux selinux=1 enforcing=1` to match common distros (this is cmdline fragment from SUSE). Maybe some tools care about this

Reply:
https://www.kernel.org/doc/html/v6.11/admin-guide/kernel-parameters.html seems it's all standard kernel parameters

Reply:
`[    0.014439] Unknown kernel command line parameters "pti=on selinux=1", will be passed to user space.`